src/cmd/sanity_test/sanity_tests.go

```diff
@@ -418,22 +452,77 @@ func permissionsIn(
 		fatal(ctx, "getting permission", err)
 	}
 
-	if len(init) > 0 {
-		maps.Copy(result, init)
+	for _, per := range pcr.GetValue() {
```

Comment:
Prefer `perm` as a shortening to `per`, please.  "Per" usually to iteration, eg: `per item in...`.  Perm is more easily understood as "permission".
```suggestion
	for _, per := range pcr.GetValue() {
```